nucliadb_vectors/src/lib.rs

```diff
@@ -37,6 +37,8 @@ pub enum VectorErr {
     WorkDelayed,
     #[error("Several writers are open at the same time ")]
     MultipleWriters,
+    #[error("Can not initialize the merger twice")]
```

Comment:
"Can not initialize the merge twice" => "Merger is already initialized"

Because a "dumb" user can try more than twice to initialize the merger :smile_cat:

Reply:
Done at [d846d76](https://github.com/nuclia/nucliadb/pull/600/commits/d846d76971f64cd8d803c4562c05718265a55ea3)